KeystoneHQ/keystone3-firmware
# Feature/add fr translation

## Explanation

Adding french translation to the data.csv




## Pre-merge check list

## How to test



There is no informations on how to run the unit tests.
A bit of documentation on that and adding a new locale could be awesome.

Anyway it should add the French language. May need a step to finish it. But can't figure it.

If I can have the next steps to finish the integration of the French language, it should add French!
